micronutrientsupport/micronutrient-support-tool
# feat: implemented mn fv tables in step 2 & 3

- Updated table in step 2 & 3 based on conversation with AB this am. 
- Prevent Errors when data returned from API is NULL. (To test: use review intervention ID 6)

To Test:

1. Inspect > Application > Clear site data 
2. Select intervention: id 5
3. View steps 1 > 2 > 3 ('Other Mn in Premix' table should be empty).
4. Return to step 1 > Add Mn to Premix > View step 2 > view step 3 (Mn should be visible in 'Other Mn in Premix' table)

![image](https://user-images.githubusercontent.com/38860838/218776464-ec4094fe-866a-4120-8a17-e15d4568e57e.png)
